src/main/java/org/junitpioneer/jupiter/resource/NewPort.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2016-2023 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter.resource;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@New(FreePort.class)
```

Comment:
I think it means that a new instance of the `FreePort` class is created, but I would like to double check as I know `@New` only as a deprecated CDI annotation (and I'm not getting any other proper search result from google 🆘). This would mean that the user always get a Socket on port zero - which is hardly what they want I guess.

Reply:
Again - NewPort was removed. The `@New` annotation is part of our resources abstract extension.

Reply:
Thanks for updating.